### PR TITLE
Main Cleanup of SANS release notes

### PR DESCRIPTION
**Pull request by sf1919**
**Description of work.**

The main refactoring the the SANS release notes ahead of release of 6.2. Includes:-

- Adding links
- Updating titles and headings
- Re-ordering information
- Spell checking

**To test:**

- Check that the links work
- Grammar/Spelling check

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by gemmaguest — CHANGES_REQUESTED** [review]
The sub-list formatting appears as a quote:

image

This can be fixed by making sure the sub-list indent lines up with the text of the main item (using spaces instead of tabs). You also need a blank line before and after the sub-list, i.e.:

image

File: docs/source/release/v6.2.0/sans.rst [hunk 1]
```diff
@@ -5,39 +5,43 @@ SANS Changes
...
-  If you have toggled any save options it will not update the selected methods until the interface is restarted to avoid interfering with the user's save selection.
-- :ref:`Q1DWeighted <algm-Q1DWeighted>` now supports kinetic data from SANS, with multiple samples instead of time bins.
+* :ref:`SANSILLIntegration <algm-SANSILLIntegration>` has new a resolution calculation option alternative to Mildner-Carpenter based on fitting horizontal size of direct beam. The fitting is handled in :ref:`SANSILLReduction <algm-SANSILLReduction>` while processing beam.
+* :ref:`LoadILLSANS <algm-LoadILLSANS>` is extended to support loading monochromatic kinetic files from the new D11.
+* :ref:`SANSILLReduction <algm-SANSILLReduction>` is extended to support processing of monochromatic kinetic runs.
+* A series of improvements have been introduced to :ref:`SANSILLAutoProcess <algm-SANSILLAutoProcess>` including
 
-Bugfixes
---------
+  * the use of the new :ref:`Stitch <algm-Stitch>` algorithm allowing for semi-automatic q binning
```
Comment (gemmaguest) [inline docs/source/release/v6.2.0/sans.rst:17]: "allowing for semi-automatic q binning" was originally a separate item in the list but it has now been rolled into the same line as the previous one. Is that a mistake?

**Review by gemmaguest — CHANGES_REQUESTED** [review]

File: docs/source/release/v6.2.0/sans.rst (hunk 1 above)
Comment (sf1919) [reply docs/source/release/v6.2.0/sans.rst:17]: Yes I think it is. Will get that sorted.

**Review by gemmaguest — APPROVED** [review]
Looks good now :+1:

Just one minor note for the future - it's good to try and avoid commit messages like "further changes" or "changes on review". You've actually got some good detail within the body of those messages about what the changes are which is really good. You could also write a short header line that is a bit more specific to what the commit contains too. Even if it's just "minor formatting changes" for example then I think that's still useful (imagine looking back at the log in a year's time - "changes from review" will no longer be relevant, and could be anything from minor tidying to significant rework! So saying "minor formatting changes" can give a better indication of what the commit contains).

**Pull request merged**